Regular-Question-Answers/teemo-attacking.cpp

```diff
@@ -0,0 +1,28 @@
+// Author: Anıl Dervişoğlu
+// Question Link: https://leetcode.com/problems/teemo-attacking/
+
+/*
+    Problem Statement:
+    Given a list of attack times and poison duration, calculate the total time the target is poisoned.
+    Consecutive attacks refresh the poison duration without stacking.
+*/
+
+class Solution {
+public:
+    int findPoisonedDuration(vector<int>& timeSeries, int duration) {
+        int totalPoisonedTime = 0; // Holds the total time Ashe is poisoned.
+
+        // Loop through the attack times starting from the second attack.
+        for (int i = 1; i < timeSeries.size(); i++) {
+            // If the current attack happens before the previous poison effect ends, add the difference.
+            if (timeSeries[i] - timeSeries[i - 1] < duration)
+                totalPoisonedTime += timeSeries[i] - timeSeries[i - 1];
+            else

```

Comment:
else branch açıklamasını yazmamışsın, "else, the time difference is larger than the poison duration so poison duration is only the duration" gibi bir açıklama ekleyebilirsin.